src/jug.sol

```diff
@@ -22,14 +22,14 @@ contract Jug is LibNote {
 
     // --- Data ---
     struct Ilk {
-        uint256 duty;
-        uint256  rho;
+        uint256 duty;  // Stability Fee      [ray]
+        uint256  rho;  // Time of last drip  [seconds]
     }
 
     mapping (bytes32 => Ilk) public ilks;
-    VatLike                  public vat;
-    address                  public vow;
-    uint256                  public base;
+    VatLike                  public vat;  // CDP Engine
+    address                  public vow;  // Debt Engine
+    uint256                  public base; // Global Stability Fee [wad]
```

Comment:
Similar comment here to above--something like "global per-second stability fee contribution" is less likely to be misinterpreted.

Also, minimum of two spaces between code and start of comment.

Reply:
👍 check on my reply with regards to `duty`